cmd/utils.go

```diff
@@ -547,3 +549,74 @@ func isFunded(ctx context.Context, caller address.Address) (bool, error) {
 	}
 	return bal.Cmp(big.NewInt(0)) > 0, nil
 }
+
+func confirmBackupExists() error {
+	backupExists := os.Getenv("GLIF_BACKUP_EXISTS")
+	if backupExists != "" {
+		return nil
+	}
+
+	as := util.AccountsStore()
+	if len(as.AccountNames()) == 0 {
+		return nil
+	}
+
+	bs := util.BackupsStore()
+	confirmedExists, err := bs.Get("confirmed-exists")
+	if err != nil {
+		return err
+	}
+
+	if confirmedExists == "true" {
```

Comment:
Is there a safer way to do this? Maybe we wrap the backup store in a call to parse this into a bool?

Reply:
This value isn't really something a user should ever edit - it will only be set by the code here, so there's really no danger it would ever be a different value than "true" or "false". I think I'd prefer to leave it as-is just to keep it simple and because it's already tested.